.travis.yml

```diff
@@ -1,11 +1,12 @@
 language: node_js
 node_js:
-  - '5'
-  - '4'
+  - '6'
+  - '8'
+  - '10'
 before_install:
   - 'npm install -g npm@latest'
```

Comment:
Should we specify a version here?

Reply:
I'd suggest we specify 8 (LTS) but you guys let me know.

Reply:
I was actually referring to the `npm@latest` in this comment, so it goes right along with the one up there ^

Reply:
@mattlubner

Reply:
Hey @MrChristofferson @mannyhenri I chose those nodejs versions for testing in TravisCI as they're the ones actively being maintained (see: https://github.com/nodejs/Release#release-schedule). I was thinking we should test on whatever's actively being maintained, but what do you think? (note that I omitted v9 as it's not LTS and gets EOL'd next month)

As for the `npm` version, I totally agree, we should specify it explicitly here. Will do that shortly! 🙂